astropy_healpix/conftest.py

```diff
@@ -37,3 +49,16 @@ def pytest_configure(config):
         # On older versions of Numpy, the unrecognized 'legacy' option will
         # raise a TypeError.
         pass
+
+# Uncomment the last two lines in this block to treat all DeprecationWarnings as
+# exceptions. For Astropy v2.0 or later, there are 2 additional keywords,
+# as follow (although default should work for most cases).
+# To ignore some packages that produce deprecation warnings on import
+# (in addition to 'compiler', 'scipy', 'pygments', 'ipykernel', and
+# 'setuptools'), add:
+#     modules_to_ignore_on_import=['module_1', 'module_2']
+# To ignore some specific deprecation warning messages for Python version
+# MAJOR.MINOR or later, add:
+#     warnings_to_ignore_by_pyver={(MAJOR, MINOR): ['Message to ignore']}
+# from astropy.tests.helper import enable_deprecations_as_exceptions  # noqa
+# enable_deprecations_as_exceptions()
```

Comment:
Remove this block - it's going to be removed in the template now that this is deprecated in astropy core

Reply:
Done.